app/firebase.js

```diff
@@ -0,0 +1,24 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAnalytics } from "firebase/analytics";
+import { getFirestore } from "firebase/firestore";
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: process.env.FIRESTORE_API_KEY,
+  authDomain: "checkedin-cf697.firebaseapp.com",
```

Comment:
looks good! we can probably consider doing staging deployments with vercel (QA/dev) and production deployments with firebase.